Donation-platform/src/main/webapp/trendingScript.js

```diff
@@ -54,7 +57,8 @@ function updateResultsOnPage(charities) {
   //resultsContainer.innerHTML += '<li>' + 'name' + '</li>';
   // add results to the page
   //for (const name of charities) {
-  for (index = 0; index < charities.length; index++) {
-    resultsContainer.innerHTML += '<li>' + charities[index] + '</li>';
+  for (index = 0; index < charities.length - 1; index += 2) {
```

Comment:
why are we jumping with 2?

Reply:
I jump by two because in the array, I store the charity name followed by the tags, both of which I print in pairs to the display.

Reply:
Added comment below. This is not the right way to store it. Use Pair<> https://www.baeldung.com/java-pairs instead

Reply:
changed the logic to return the entire JSON rather than extracting fields.